portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,21 +38,50 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  static final int LOAD_SIZE = 5;
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  private final ArrayList<Comment> comments = new ArrayList<>();
+  private FetchOptions fetchOptions;
+  private ArrayList<Comment> comments;
+  private String cursor;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    this.comments = new ArrayList<>();
```

Comment:
Can you just make this a local variable since it's not used outside of this function?